examples/vulkan_example/imgui_impl_glfw_vulkan.cpp

```diff
@@ -236,10 +236,10 @@ void ImGui_ImplGlfwVulkan_RenderDrawLists(ImDrawData* draw_data)
         VkMappedMemoryRange range[2] = {};
         range[0].sType = VK_STRUCTURE_TYPE_MAPPED_MEMORY_RANGE;
         range[0].memory = g_VertexBufferMemory[g_FrameIndex];
-        range[0].size = vertex_size;
+        range[0].size = VK_WHOLE_SIZE;
         range[1].sType = VK_STRUCTURE_TYPE_MAPPED_MEMORY_RANGE;
         range[1].memory = g_IndexBufferMemory[g_FrameIndex];
-        range[1].size = index_size;
+        range[1].size = VK_WHOLE_SIZE;
```

Comment:
We should also align vkMapMemory to nonCoherentAtomSize.

Note that we are not always using the full size of the vertex and index buffer. This could be fixed by using vertex_buffer_size and index_buffer_size (guaranteed to be multiples of nonCoherentAtomSize) when we map and flush the buffers.